modules/21_tax/on/datainput.gms

```diff
@@ -139,9 +139,24 @@ $offdelim
 *** converted to T$/TWyr   
 p21_tau_xpres_tax(ttot,regi,"peoil")$(ttot.val ge 2005) = p21_tau_xpres_tax(ttot,regi,"peoil") * sm_DpGJ_2_TDpTWa;
 *LB* use 0 for all regions as default
-p21_tau_xpres_tax(ttot,regi,all_enty) = 0;  
+p21_tau_xpres_tax(ttot,regi,all_enty) = 0;
+
+
+*** -------------------------Primary Energy Tax--------------------------
+*** initialize tax to zero
+p21_tau_pe_tax(ttot,regi,all_enty) = 0;
+
+*PW* choose tax path for gas in Germany
```

Comment:
Well, Renato is right. We have this convention of only using regional names in the regipol module. So this should be moved to ``regipol/regiCarbonPrice/datainput.gms``

Reply:
Alternatively, you could try to define it without a specific region and specify it in the config file as only applying to Germany.

Reply:
> Alternatively, you could try to define it without a specific region and specify it in the config file as only applying to Germany.

Ideally, any switch that include regional specification should also belong to the regipol module. This is not always possible because how the code is currently organized, but we should aim for that as much as we can.

Reply:
Thanks for the comments, will move it tomorrow!